src/interactive/documentation.ts

```diff
@@ -0,0 +1,227 @@
+    )
+    setPanelContext()
+    vscode.window.registerWebviewPanelSerializer(viewType, new DocumentationPaneSerializer())
+}
+
+function showDocumentationPane() {
+    if (panel === undefined) {
+        panel = createDocumentationPanel()
+        setPanelSubscription(panel)
+    }
+    if (panel !== undefined && !panel.visible) {
+        panel.reveal()
+    }
+}
+
+function createDocumentationPanel() {
+    return vscode.window.createWebviewPanel(viewType, 'Julia Documentation Pane',
+        {
+            preserveFocus: true,
+            viewColumn: vscode.ViewColumn.Beside,
+        },
+        {
+            enableFindWidget: true,
+            // retainContextWhenHidden: true, // comment in if loading is slow, while there would be high memory overhead
+            enableScripts: true,
+        }
+    )
+}
+
+class DocumentationPaneSerializer implements vscode.WebviewPanelSerializer {
+    async deserializeWebviewPanel(deserializedPanel: vscode.WebviewPanel, state: any) {
+        panel = deserializedPanel
+        setPanelSubscription(panel)
+        const { inner } = state
+        const html = createWebviewHTML(inner)
+        _setHTML(html)
+    }
+}
+
+function setPanelSubscription(panel: vscode.WebviewPanel) {
+    panel.onDidChangeViewState(({ webviewPanel }) => {
+        setPanelContext(webviewPanel.active)
+    })
+    panel.onDidDispose(() => {
+        setPanelContext(false)
+        if (messageSubscription !== undefined) {
+            messageSubscription.dispose()
+        }
+        panel = null
+    })
+    setPanelContext(true)
+}
+
+function setPanelContext(state: boolean = false) {
+    setContext(panelActiveContextKey, state)
+}
+
+const LS_ERR_MSG = `
+Error: Julia Language server is not running.
+Please wait a few seconds and try again once the \`Starting Julia Language Server...\` message in the status bar is gone.
+`
+async function showDocumentation() {
+    // telemetry.traceEvent('command-showdocumentation')
+    const inner = await getDocumentation()
+    setDocumentation(inner)
+}
+
+async function getDocumentation(): Promise<string> {
+    const editor = vscode.window.activeTextEditor
+    const selection = editor.selection
+    const position = new vscode.Position(selection.start.line, selection.start.character)
+
+    return await withLanguageClient(
+        async languageClient => {
+            return languageClient.sendRequest('julia/getDocAt', getVersionedParamsAtPosition(editor, position))
+        },
+        err => {
+            vscode.window.showErrorMessage(LS_ERR_MSG)
+            return ''
+        }
+    )
+}
+
+function setDocumentation(inner: string) {
+    if (!inner) { return }
+    forwardStack = [] // initialize forward page stack for manual search
+    showDocumentationPane()
+    const html = createWebviewHTML(inner)
+    _setHTML(html)
+}
+
+function createWebviewHTML(inner: string) {
+    const darkMode: boolean = vscode.workspace.getConfiguration('julia.documentation').darkMode
```

Comment:
Can't we use `vscode.window.activeColorTheme.kind == ColorThemeKind.Dark` here? I'm not a big fan of this being a setting.

Reply:
How/where did you find the API ??? I tried many times to find such an API and have been unable to find that ...
That should work, and yes I also really don't like to make this being a setting, of course !

Reply:
It works, thanks for the suggestion.